controllers/doguInstallManager.go

```diff
@@ -27,6 +26,8 @@ import (
 )
 
 const k8sDoguOperatorFieldManagerName = "k8s-dogu-operator"
+const k8sNginxIngressDoguName = "nginx-ingress"
+const k8sNginxStaticDoguName = "nginx-static"
```

Comment:
🚫 **[checkstyle]** <unused> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>const `k8sNginxStaticDoguName` is unused
<!-- __reviewdog__:ChBhZDI1ZmE4MTJlMzJjNWYwEgpjaGVja3N0eWxl -->